### ISSUE TITLE
Any news regarding the release?

### ISSUE THREAD

**Issue by sistein** [nello-one/nello-pi#3]
Any news regarding the release?

**Comment by codeclinic** [comment]
Eerily quiet...

**Comment by carloscae** [comment]
i'm about to throw this crap in the garbage and go for the Nuki solution.
for months now it's just something blinking on my wall, that does nothing useful.

**Comment by pattyland** [comment]
@carloscae Nello is working fine for months now, what's your problem?

**Comment by carloscae** [comment]
> @carloscae Nello is working fine for months now, what's your problem?

no app push notifications since march?

**Comment by pattyland** [comment]
Nello is integrated into my Home Automation System and I never used the official App (like all other apps). I get notified via Telegram (Push to smartphone) and Alexa (voice controlled when I’m at home) which is working great and reliable

**Comment by Morpheus2002** [comment]
@pattyland how did you managed to get third party service notifications? 
Push is not working for me for a long time too.

**Comment by pattyland** [comment]
With their official API? https://nellopublicapi.docs.apiary.io/

I'm using it together with Home Assistant to integrate it with my other hardware like Alexa and Siri: https://www.home-assistant.io/integrations/nello/

**Comment by jeboehm** [comment]
Personally, I've problems creating new access tokens on https://auth.nello.io/admin for a while now. That page seems to be down.

**Comment by Morpheus2002** [comment]
@pattyland as @jeboehm mentioned the API page is down/not loading. So nobody can access it anymore.

**Comment by CaptInsano** [comment]
I see a few preowned Nello Ones being sold cheaply online. Is there currently any way to enable these with HomeAssistant? 

I see above that [https://auth.nello.io/admin](url) is down, it that required?

**Comment by sistein** [comment]
I'm unable to use nello right now. Opening the door fails, loading the tab Users fails, and loading activity too. 

Anyone else having these problems?

**Comment by sleepyarmistice** [comment]
Yes, the app intermittently doesn’t connect for me. Don’t forget your keys!

**Comment by sistein** [comment]
@nello-one it would be great if you can give us some informations regarding the actual status of the usage of nello. Especially regarding the future. 

In the last weeks some updates would have been nice.

**Comment by carloscae** [comment]
> With their official API? https://nellopublicapi.docs.apiary.io/
> 
> I'm using it together with Home Assistant to integrate it with my other hardware like Alexa and Siri: https://www.home-assistant.io/integrations/nello/

yeah. used it for a while but the hacked integration was always very buggy to me creating numerous errors and requests. there was never a proper API integration built for HASS.

and guys, honestly, this crap is going nowhere. 
better count our losses and get a Nuki Opener.

**Comment by CaptInsano** [comment]
Apparently the Nuki Opener is not working with homeassistant:

Source: open issue since Sept: 
https://github.com/home-assistant/home-assistant/issues/26447

Further discussion:
https://community.home-assistant.io/t/locumi-labs-gmbh-nello-io-insolvent/128793

**Comment by codeclinic** [comment]
> Apparently the Nuki Opener is not working with homeassistant:

Nuki is nothing to do with Nello. They are 2 separate entities.

**Comment by CaptInsano** [comment]
I am aware of that, just commenting as to how Nello being broken is not solved by Nuki Opener as per prev comment by @carloscae

**Comment by carloscae** [comment]
> Apparently the Nuki Opener is not working with homeassistant:
> 
> Source: open issue since Sept:
> [home-assistant/home-assistant#26447](https://github.com/home-assistant/home-assistant/issues/26447)
> 
> Further discussion:
> https://community.home-assistant.io/t/locumi-labs-gmbh-nello-io-insolvent/128793

This is more of an issue with HASS developers having resources to tackle the problem.
Integration is not official, Nuki only provides the API. Nuki still works on all official integrations and its own app.

You guys can support HASS development by subscribing to Nabu Casa.

**Comment by codeclinic** [comment]
> 
> 
> > Apparently the Nuki Opener is not working with homeassistant:
> 
> Nuki is nothing to do with Nello. They are 2 separate entities.



> 
> 
> I am aware of that, just commenting as to how Nello being broken is not solved by Nuki Opener as per prev comment by @carloscae

Then it's not relevant to post here. This is an issue thread for Nello. Or should we derail this thread and start listing every device that works with Home assistant? How would that help exactly?

**Comment by carloscae** [comment]
> > > Apparently the Nuki Opener is not working with homeassistant:
> > 
> > 
> > Nuki is nothing to do with Nello. They are 2 separate entities.
> 
> > I am aware of that, just commenting as to how Nello being broken is not solved by Nuki Opener as per prev comment by @carloscae
> 
> Then it's not relevant to post here. This is an issue thread for Nello. Or should we derail this thread and start listing every device that works with Home assistant? How would that help exactly?

It doesn't hurt either. 
This project is dead. They sold their assets and now it doesn't belong to Nello anymore. 

People lost time and money on this and you're worried that a Github issue titled "Any news on this?" is derailing after 2 months of zero replies from Nello.

**Comment by codeclinic** [comment]
No, the company was sold as a whole and still exists, in what form we still have to see. Regardless, hijacking a thread just makes it significantly less likely that an issue will get resolved since it makes genuine discussion near impossible.

**Comment by CaptInsano** [comment]
> Then it's not relevant to post here. This is an issue thread for Nello. Or should we derail this thread and start listing every device that works with Home assistant? How would that help exactly?

Ah man,  I was just trying to help anyone who might rush out to buy another product that would also not work at present with homeassistant. 

It is not like this is an actual technical bug report, it is titled "Any news regarding the release?"

**Comment by carloscae** [comment]
Again “hijacking” the thread for a PSA:

https://www.golem.de/news/smarter-tueroeffner-nello-one-oeffnet-fremden-ungefragt-die-tuer-2001-146368.html

Seems like Nello GPS function that allows you to open the door with a ring when you arrive home is broken. 

Now it opens the door to everyone who rings your door. Fix will take a while and they recommend via support emails to disable the functionality. 

I’m removing it from my door today.

**Comment by sistein** [comment]
This problem exists already for several weeks. Unfortunaly I was unable to contact ANYONE. 

In the end I pulled the cable closing the circuit for dooropening. No information by Nello btw! 

Its a shame. Would have been better to give a open source solution.

**Comment by codeclinic** [comment]
It looks like they got bought out with the hope of growing the business but instead the new owners just ripped out the assets and tech for their own benefit then left nello and its customers to die in a ditch.




On Fri, Jan 31, 2020 at 5:25 PM +0100, "sistein" <notifications@github.com> wrote:












This problem exists already for several weeks. Unfortunaly I was unable to contact ANYONE.


In the end I pulled the cable closing the circuit for dooropening. No information by Nello btw!


Its a shame. Would have been better to give a open source solution.

**Comment by Morpheus2002** [comment]
I really hate to say it, but now all Nello users have one more reason to switch to Nuki Opener. I already did it and it's really working well.

**Comment by sistein** [comment]
@nello-one is there someone? 

@Morpheus2002 My next one will be self-build. Like I should have had it done before...

**Comment by carloscae** [comment]
> @Morpheus2002 My next one will be self-build. Like I should have had it done before...

I’m not smart enough to do something like this. Would love to follow on your project when it comes to it.
